README.md

```diff
@@ -36,7 +36,7 @@ npm run serve
 ### Docker Build
 ```
 npm install
-npm run serve
+npm run build
```

Comment:
You shouldn't use `build` in my mind. Build should active auth from my understanding.

Reply:
`npm run serve` just start this project for development, will not generate any resource in folder `./dist`.

Reply:
Get your point. @JaredTan95 consider this docker becomes the test env only, could you consider to change to as serve mode without nginx, just nodejs?